components/raftstore/src/store/msg.rs

```diff
@@ -916,6 +911,10 @@ where
     AwakenRegions {
         abnormal_stores: Vec<u64>,
     },
+
```

Comment:
```suggestion
// Developers should be cautious when adding new `StoreMsg` variants in the future to avoid unintended consequences.
```

Recommend to add some comments here as remainder:

Reply:
There is already a remainder in `discriminant`